src/Func.ls

```diff
@@ -12,8 +12,15 @@ fix = (f) ->
 
 over = (f, g, x, y) --> f (g x), (g y)
 
-#? memoize, wrap
+memoize = (f) ->
+  memo = {}
+  ->
+    args = Array::slice.call &, 0
+    key = args.join ''
```

Comment:
I pinched this technique from lodash, but you're right. Unfortunately, though, `Map` isn't available in the stable version of node. I'll see if I can come up with something a little better.